lib/read.js

```diff
@@ -217,6 +238,15 @@ class ReadSql extends AdapterRead {
         } else {
             query = query.offset(this.offsetCount);
         }
+        
+        if (this.sortBorderValue) {
+            query = query.where(
+                this.sortFields[0].field,
+                this.sortFields[0].direction === 'desc' ? '<=' : '>=',
```

Comment:
So the way this works is that each result set consists of all values of the border value. For example, lets say the `borderVal`s were `1,2,2,3,3,4,5` and the `fetchSize` is `4` then only `1,2,2` would be returned: the last value becomes the border value none of those will be in the result set. Then the next result set is `>=3`.  In other words, points with the border value are not included in result set. If the entire fetch has the same border value then we throw an error.